docs/source/tutorials/exploration.rst

```diff
@@ -95,6 +95,7 @@ configuration by simply printing it.
    sim = get_disease_model_simulation()
 
    del sim.configuration['input_data']
+   del sim.configuration['stratification']['excluded_categories']
```

Comment:
This key was throwing errors in the expected `print(sim.configuration)` test below. The actual type of `sim.configuration.stratification.excluded_categories` is an emtpy `LayeredConfigTree` which I just couldn't get to work (I tried None, an empty string, literally nothing, and LayeredConfigTree() and none of them worked).

Reply:
I actually was able to get this test to pass if I set the default to None (instead of an empty dict which then gets converted to a LayeredConfigTree) but that broke a bunch of tests in vph when trying to `.update` the value b/c you cannot update a ConfigNode.